### PR TITLE
feat(mongodb): Add MongoDB extension

### PR DESCRIPTION
**Pull request by vymarkov**

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -883,6 +883,12 @@
       "repository": "https://github.com/mitchdenny/ecdc",
       "version": "1.4.0"
     },
+    {
+      "id": "mongodb.mongodb-vscode",
+      "repository": "https://github.com/mongodb-js/vscode",
+      "version": "0.4.0",
+      "checkout": "v0.4.0"
```
Comment (jankeromnes) [inline extensions.json:890]: Same remark as in https://github.com/open-vsx/publish-extensions/pull/275#discussion_r567922516: 

```suggestion
      "repository": "https://github.com/mongodb-js/vscode/releases/download/v0.4.0/mongodb-vscode-0.4.0.vsix",
      "version": "0.4.0"
```

**Review by jankeromnes — APPROVED** [review]
Awesome, many thanks! 👍

**Review by jankeromnes — APPROVED** [review]
🚀

**Comment by jankeromnes** [comment]
@vymarkov Whoops, auto-publishing failed with:

```
Running: git clone --recurse-submodules https://github.com/mongodb-js/vscode/releases/download/v0.4.0/mongodb-vscode-0.4.0.vsix /tmp/repository
[FAIL] Could not process extension: {
  "id": "mongodb.mongodb-vscode",
  "repository": "https://github.com/mongodb-js/vscode/releases/download/v0.4.0/mongodb-vscode-0.4.0.vsix",
  "version": "0.4.0"
}
Error: Command failed: git clone --recurse-submodules https://github.com/mongodb-js/vscode/releases/download/v0.4.0/mongodb-vscode-0.4.0.vsix /tmp/repository
Cloning into '/tmp/repository'...
remote: Not Found
fatal: repository 'https://github.com/mongodb-js/vscode/releases/download/v0.4.0/mongodb-vscode-0.4.0.vsix/' not found
```

Looks like a broken configuration (should be `"download"` instead of `"repository"`).

In the future, please feel free to use the helper script, which will almost always do the right thing. For example:

```
$ node add-extension https://github.com/mongodb-js/vscode --checkout
```

will see that there is an official `.vsix` available and use that:

```diff
+    {
+      "id": "mongodb.mongodb-vscode",
+      "repository": "https://github.com/mongodb-js/vscode/releases/download/v0.4.0/mongodb-vscode-0.4.0.vsix",
+      "version": "0.4.0"
+    },
```

**Comment by vymarkov** [comment]
Awesome,thank you 🤘

**Pull request merged**